tests/testthat/test-radsl-alt-args.R

```diff
@@ -0,0 +1,24 @@
+options(pillar.print_max = Inf, pillar.width = Inf)
+
+set.seed(1)
+
+test_that("radsl produces expected values with change of sex percentage", {
```

Comment:
I do not think we need to test snapshots for this. It is enough to pull the result and check it against set value with an acceptable tolerance.

The PR looks very good otherwise! ;) Thank you